src/main/java/net/javadiscord/javabot/systems/user_commands/format_code/FormatCodeCommand.java

```diff
@@ -49,7 +46,11 @@ public FormatCodeCommand() {
 								.addChoice("SQL", "sql")
 								.addChoice("Swift", "swift")
 								.addChoice("TypeScript", "typescript")
-								.addChoice("XML", "xml")
+								.addChoice("XML", "xml"),
+						new OptionData(OptionType.STRING,"auto-indent","The type of indentation applied to the message, does not automatically indent if left blank.",false)
+								.addChoice("Four_Spaces","four")
```

Comment:
These values do not work with `IndentationHelper.IndentationType.valueOf`. They would need to exactly match the enum constants.